### PR TITLE
Add a way to print a list of sprite

### PR DESCRIPTION
**Pull request by fjtrujy**
## Description
This PR does:
- Fix wrong forced alignment, I think they aren't needed as the first value in the union defines the alignement
- Create some other inline functions to deal more easily with the colors
- Add `gsKit_prim_list_sprite_gouraud_3d` for allowing to draw a list of sprites
- Update `gsKit_clear` for using `gsKit_prim_list_sprite_gouraud_3d` better.

Cheers

### CODE REVIEW

File: ee/gs/include/gsInit.h [hunk 1]
```diff
@@ -970,58 +970,65 @@ typedef union {
...
-} __attribute__((packed,aligned(8))) gs_uv;
+} __attribute__((packed)) gs_uv;
 
 typedef union {
 	u128 stq;
 	struct {
 		gs_stq_t st;
 		u64 tag;
 	};
-} __attribute__((packed, aligned(8))) gs_stq;
```
Comment (fjtrujy) [inline ee/gs/include/gsInit.h:1024]: It had wrong alignment it should be 16 instead

File: ee/gs/include/gsInit.h [hunk 2]
```diff
@@ -970,58 +970,65 @@ typedef union {
...
+} __attribute__((packed)) gs_rgbaq;
 
 typedef union {
 	u128 uv;
 	struct {
 		u64 coord;
 		u64 tag;
 	};
-} __attribute__((packed,aligned(8))) gs_uv;
+} __attribute__((packed)) gs_uv;
```
Comment (fjtrujy) [inline ee/gs/include/gsInit.h:1023]: It had wrong alignment it should be 16 instead, but let the compiler to take it based on `u128`

File: ee/gs/include/gsInit.h [hunk 3]
```diff
@@ -970,58 +970,65 @@ typedef union {
...
+} __attribute__((packed)) gs_xyz2;
 
 typedef union {
 	u128 rgbaq;
 	struct {
 		gs_rgbaq_t color;
 		u64 tag;
 	};
-} __attribute__((packed,aligned(8))) gs_rgbaq;
+} __attribute__((packed)) gs_rgbaq;
```
Comment (fjtrujy) [inline ee/gs/include/gsInit.h:1015]: It had wrong alignment it should be 16 instead, but let the compiler to take it based on `u128`

**Review by uyjulian — APPROVED** [review]
Lgtm

**Pull request merged**